packages/cicero-core/test/data/verifying-template-signature/helloworldstateSigned/package.json

```diff
@@ -1 +1 @@
-{"name":"helloworldstate","displayName":"Hello World State","version":"0.15.0","description":"This is the stateful Hello World of Accord Project Templates. Executing the clause will simply echo back the text that occurs after the string `Hello` prepended to text that is passed in the request along with the number of times the clause has been called.","author":"Accord Project","license":"Apache-2.0","accordproject":{"template":"clause","cicero":"^0.24.0","runtime":"ergo"},"devDependencies":{"cucumber":"^5.1.0"},"scripts":{"test":"cucumber-js test -r .cucumber.js"},"keywords":["hello","world","greet"]}
\ No newline at end of file
+{"name":"helloworldstate","displayName":"Hello World State","version":"0.15.0","description":"This is the stateful Hello World of Accord Project Templates. Executing the clause will simply echo back the text that occurs after the string `Hello` prepended to text that is passed in the request along with the number of times the clause has been called.","author":"Accord Project","license":"Apache-2.0","accordproject":{"template":"clause","cicero":"^0.24.0","runtime":"cicero"},"devDependencies":{"cucumber":"^5.1.0"},"scripts":{"test":"cucumber-js test -r .cucumber.js"},"keywords":["hello","world","greet"]}
```

Comment:
I assume the cucumber stuff is now dead?

Reply:
Yep, cicero-test is gone

Reply:
I'll remove in a future PR.